test/spark/app/.gitignore

```diff
@@ -0,0 +1,34 @@
+*.class
+*.log
+*.zip
+*.gz
+*.jpeg
+*.jpg
+*.png
+*~
+
+# sbt specific
+.settings/
+.cache/
+.history/
+.lib/
+dist/*
+target/
+lib_managed/
+src_managed/
+project/boot/
+project/plugins/project/
+.project
+.classpath
+.cache
+
+# Scala-IDE specific
+.scala_dependencies
+.worksheet
+
+# IntelliJ IDEA specific
+**/.idea/
+**/target/
+**/*.iml
```

Comment:
Editor-specific files don't belong in any `.gitignore` file, especially not in an innermost level.  Put it in your private `~/.config/git/ignore` (or wherever MacOS puts app-specific config files), not here.

Because after that you get a VI user adding `*.swp`, then `*.sw[a-z]` because VI uses additional swap files, and files start vanishing from commits :shrug: .

Reply:
Removed